docs/lms-connection.md

```diff
@@ -20,6 +20,7 @@ based on a regex match on the URL of the course.
 LMS_BACKENDS=[
     {
         "BACKEND": "richie.apps.courses.lms.edx.TokenEdXLMSBackend",
+        "JS_BACKEND": "EDX_HAWTHORN",
```

Comment:
It looks like a constant name?
```suggestion
        "JS_BACKEND": "edx-hawthorn",
```

Reply:
That was my purpose. I use this convention for enum types in ts.